TicTacToeApp/Game.swift

```diff
@@ -0,0 +1,36 @@
+
+public class Game {
```

Comment:
Similar overall comments as to `Board`. Small functions, short interface. Not too many private functions.